### PR TITLE
mchx campaign banner and mint on gallery

### PR DESCRIPTION
**Pull request by pvicensSpacedev**
### Summary of Changes

- Added the banner for mchx mint in mobile (web still in progress)
- Added pin notification UI (have to properly get the data from the backend)
- Added Mint on gallery button (opens campaign bottom sheet) if not claimed and mint on highlight if already minted

### Demo or Before/After Pics
Screenshot 2024-04-11 at 17 13 22
Screenshot 2024-04-11 at 17 13 30
Screenshot 2024-04-11 at 17 13 26
Screenshot 2024-04-11 at 17 13 50
Screenshot 2024-04-11 at 15 44 07
Screenshot 2024-04-11 at 17 13 38

### Edge Cases

List any common edge cases that you have considered and tested.

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

**Comment by kaitoo1** [comment]
we should show the Gallery G logo here instead of Foundation's
CleanShot 2024-04-12 at 11 25 04

**Comment by kaitoo1** [comment]
and if it's "Mint on Gallery" I think we can toss the diagonal arrow since that is saying it's an external site

**Comment by kaitoo1** [comment]
otherwise looks great! banner looks nice too 🔥

File: apps/mobile/src/components/MintLinkButton.tsx [hunk 1]
```diff
@@ -113,10 +125,17 @@ export function MintLinkButton({
...
       };
     } else if (mintProviderType === 'Highlight') {
-      return {
-        buttonText: 'mint on highlight',
-        icon: <HighlightIcon width={size === 'sm' ? 16 : 24} height={size === 'sm' ? 16 : 24} />,
-      };
+      if (isRadiance && !claimCode) {
+        return {
+          buttonText: 'mint on gallery',
+          icon: <FoundationIcon width={size === 'sm' ? 16 : 24} height={size === 'sm' ? 16 : 24} />,
```
Comment (kaitoo1) [inline apps/mobile/src/components/MintLinkButton.tsx:131]: lets use the Gallery G icon here

Comment (kaitoo1) [reply apps/mobile/src/components/MintLinkButton.tsx:131]: my bad realized I didnt like the design in the ticket
we want it to look like:
CleanShot 2024-04-12 at 11 29 01

and inverse on dark mode

**Review by kaitoo1 — APPROVED** [review]

**Review by Robinnnnn — APPROVED** [review]

**Pull request merged**